Makefile

```diff
@@ -0,0 +1,42 @@
+IMAGE_REPO ?= quay.io/nmstate
+IMAGE_TAG ?= latest
```

Comment:
We have be using different tag for different images

```make
MANAGER_IMAGE_TAG ?= latest
```

Reply:
Done